librz/core/cmd_analysis.c

```diff
@@ -5325,6 +5399,10 @@ static void cmd_analysis_esil(RzCore *core, const char *input) {
 		rz_analysis_op_fini(&aop);
 		break;
 	}
+	case 'z': { // "aez"
+		cmd_analysis_rzil(core, input + 1);
```

Comment:
Was anything already decided whether the `aez` commands should be implemented in newshell directly? @ret2libc @XVilka

Reply:
@thestr4ng3r I think it's fine to have it as an oldshell in this PR to avoid irrelevant delays. They can be converted later in https://github.com/rizinorg/rizin/issues/1533

Reply:
ack